save-sandbox/build.gradle.kts

```diff
@@ -50,4 +52,13 @@ dependencies {
     implementation(libs.spring.boot.starter.security)
     implementation(libs.spring.security.core)
     implementation(libs.save.common.jvm)
+
+    implementation(libs.ktlint.core)
+    implementation(libs.ktlint.rulesets.standard)
+    implementation("org.cqfn.diktat:diktat-common:$diktatVersion") {
```

Comment:
I don't really like the idea of putting diktat-demo service in the same monorepo as save-cloud. At least let's make it a separate module and not mix with sandbox. In the future we could have a demo service with pluggable modules for different modes.

My reason: we had a lot of problems with dependency versions in diktat-demo, because diktat/ktlint require a very specific version of kotlin-compiler-embeddable, which should be the same as version of Kotlin compiler and Kotlin dependencies. In this case this having all versions mixed together will force us either to update save-cloud less frequently or occasionally break diktat-demo.

I would imagine something like what spotless does: a common facade which loads different tools in isolated classloaders.

Reply:
The next phase includes moving all demo staff to external spring service.